src/norm/postgres.nim

```diff
@@ -316,3 +316,28 @@ template transaction*(dbConn; body: untyped): untyped =
 
     raise
 
+
+# One-to-Many Fetching
+proc selectOneToMany*[O: Model, M: Model](dbConn; oneEntry: O, relatedEntries: var seq[M]) =
+  const foreignKeyFieldName: string = O.getRelatedFieldNameOn(M)
```

Comment:
1. Does this still apply after the revision? 
2. I do this mostly in my own codebases since intellisense in vscode takes forever and I want to be able to see the types at glance if it's not immediately painfully obvious what they are. I removed them from "manyTableNames" and when defining "sqlCondition" since one could argue it *should* be obvious what type they return.
3. I covered this case by changing this proc to also receive a `foreignKeyFieldName` parameter and an overload that infer that parameter for me in a nieche scenario. One could argue if that nieche scenario is even desired or not. If not, I could throw out that overload as well as `getRelatedFieldNameTo`.